static/js/reducers/moderation.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import {
+  SET_MODERATING_COMMENT,
+  CLEAR_MODERATING_COMMENT
+} from "../actions/moderation"
+
+import type { Action } from "../flow/reduxTypes"
+import type { CommentInTree } from "../flow/discussionTypes"
+
+export type ModerationState = {
+  comment: ?CommentInTree
+}
+
+export const INITIAL_UI_STATE: ModerationState = {
```

Comment:
should this be called something like `INITIAL_MODERATION_STATE`?

Reply:
Ooops you caught my copy/paste!